pkg/validatePoliciesYaml/validatePoliciesYaml_test.go

```diff
@@ -53,11 +71,17 @@ func TestValidatePoliciesYaml(t *testing.T) {
 	assertValidationResult(t, missingPolicyRules, "./test_fixtures/missingPolicyRules.yaml", nil)
 	assertValidationResult(t, missingApiVersion, "./test_fixtures/missingApiVersion.yaml", errors.New("found errors in policies file ./test_fixtures/missingApiVersion.yaml:\n(root): missing properties: 'apiVersion'"))
 	assertValidationResult(t, missingPolicyName, "./test_fixtures/missingPolicyName.yaml", errors.New("found errors in policies file ./test_fixtures/missingPolicyName.yaml:\n(root)/policies/0: missing properties: 'name'"))
+	assertValidationResult(t, missingSchema, "./test_fixtures/missingSchema.yaml", errors.New("invalid policy file: rule identifier \"PODDISRUPTIONBUDGET_DENY_ZERO_VOLUNTARY_DISRUPTION\" has no associated schema"))
+	assertValidationResult(t, multipleDefaultPolicy, "./test_fixtures/multipleDefaultPolicy.yaml", errors.New("invalid policy file: multiple policies are set to default"))
 	assertValidationResult(t, wrongApiVersion, "./test_fixtures/wrongApiVersion.yaml", errors.New("found errors in policies file ./test_fixtures/wrongApiVersion.yaml:\n(root)/apiVersion: value must be \"v1\""))
 
 	// customRule
 	assertValidationResult(t, customRuleMissingIdentifier, "./test_fixtures/customRuleMissingIdentifier.yaml", errors.New("found errors in policies file ./test_fixtures/customRuleMissingIdentifier.yaml:\n(root)/customRules/0: missing properties: 'identifier'"))
 	assertValidationResult(t, customRuleMissingName, "./test_fixtures/customRuleMissingName.yaml", errors.New("found errors in policies file ./test_fixtures/customRuleMissingName.yaml:\n(root)/customRules/0: missing properties: 'name'"))
 	assertValidationResult(t, customRuleMissingDefaultMessageOnFailure, "./test_fixtures/customRuleMissingDefaultMessageOnFailure.yaml", errors.New("found errors in policies file ./test_fixtures/customRuleMissingDefaultMessageOnFailure.yaml:\n(root)/customRules/0: missing properties: 'defaultMessageOnFailure'"))
 	assertValidationResult(t, customRuleValidSchema, "./test_fixtures/customRuleValidSchema.yaml", nil)
+	assertValidationResult(t, customRuleInvalidSchema, "./test_fixtures/customRuleInvalidSchema.yaml", errors.New("invalid policy file: rule \"PODDISRUPTIONBUDGET_DENY_ZERO_VOLUNTARY_DISRUPTION\" has schema error: has a primitive type that is NOT VALID -- given: /arrayi/ Expected valid values are:[array boolean integer number null object string]"))
+	assertValidationResult(t, customRuleInvalidJsonSchema, "./test_fixtures/customRuleInvalidJsonSchema.yaml", errors.New("invalid policy file: rule \"PODDISRUPTIONBUDGET_DENY_ZERO_VOLUNTARY_DISRUPTION\" has schema error: Invalid type. Expected: array of schemas, given: definitions"))
```

Comment:
We want the error to be:
"found errors in policies file ./test_fixtures/customRuleInvalidJsonSchema.yaml:\n(root)/customRules/1/schema: Invalid type. Expected: array of schemas, given: definitions"

note that the "1" in "customRules/1" is dynamic